stl/inc/optional

```diff
@@ -226,13 +225,13 @@ public:
 
     template <class... _Types, enable_if_t<is_constructible_v<_Ty, _Types...>, int> = 0>
     constexpr explicit optional(in_place_t, _Types&&... _Args) noexcept(
-        noexcept(_Mybase(in_place, _STD forward<_Types>(_Args)...)))
+        is_nothrow_constructible_v<_Mybase, in_place_t, _Types...>)
         : _Mybase(in_place, _STD forward<_Types>(_Args)...) {}
 
     template <class _Elem, class... _Types,
         enable_if_t<is_constructible_v<_Ty, initializer_list<_Elem>&, _Types...>, int> = 0>
     constexpr explicit optional(in_place_t, initializer_list<_Elem> _Ilist, _Types&&... _Args) noexcept(
-        noexcept(_Mybase(in_place, _Ilist, _STD forward<_Types>(_Args)...)))
+        is_nothrow_constructible_v<_Mybase, in_place_t, initializer_list<_Elem>, _Types...>)

```

Comment:
I think these are all my fault:
```suggestion
        is_nothrow_constructible_v<_Mybase, in_place_t, initializer_list<_Elem>&, _Types...>)
```

Reply:
(I suppose `in_place_t` should strictly be `const in_place_t&`, but I don't think it's worth the hit to readability to be _that_ correct when we're constructing a type that we control and therefore hopefully won't do anything silly.)